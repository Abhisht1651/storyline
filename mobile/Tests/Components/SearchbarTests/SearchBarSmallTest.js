import { shallow } from 'enzyme'
import React from 'react'

import { TextInput } from 'react-native'
import SearchBarSmall from '../../../App/Components/Searchbars/SearchBarSmall'

describe('SearchBarSmall', () => {
  it('should match to snapshot', () => {
    const component = shallow(<SearchBarSmall />)
    expect(component).toMatchSnapshot()
  })

  it('SearchBarDefault onChangetText', () => {
    const component = shallow(<SearchBarSmall />)
    component.find(TextInput).prop('onChangeText')('Random')
    expect(component.find(TextInput).prop('value')).toEqual('Random')
  })
})
