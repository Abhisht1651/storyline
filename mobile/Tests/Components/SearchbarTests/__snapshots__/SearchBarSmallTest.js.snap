// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBarSmall should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5F8F8",
      "borderRadius": 29.57142857142857,
      "flexDirection": "row",
      "height": 57.5,
      "width": 502,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 26.285714285714285,
        "justifyContent": "center",
        "marginTop": 3.2857142857142856,
        "width": 64,
      }
    }
  >
    <SvgMock
      height="100%"
    />
  </View>
  <Component
    allowFontScaling={true}
    autoCorrect={false}
    keyboardType="default"
    onChangeText={[Function]}
    placeholder="Search"
    placeholderTextColor="#8F8FAF"
    rejectResponderTermination={true}
    selectionColor="#7B46E4"
    spellCheck={false}
    style={
      Object {
        "color": "#1F1F60",
        "fontFamily": "BananaGrotesk-Light",
        "fontSize": 27,
        "paddingBottom": 11.5,
        "paddingTop": 13.142857142857142,
        "width": "70%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
</View>
`;
